SenlaApp/SenlaApp/CalculatorViewController.swift

```diff
@@ -282,7 +329,102 @@ extension CalculatorViewController {
             stackViewFirstLine.widthAnchor.constraint(equalTo: view.widthAnchor, constant:  -50),
             stackViewFirstLine.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.09),
             stackViewFirstLine.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            stackViewFirstLine.bottomAnchor.constraint(equalTo: stackViewSecondLine.topAnchor, constant: -10)
+            stackViewFirstLine.bottomAnchor.constraint(equalTo: stackViewSecondLine.topAnchor, constant: -10),
+            
+            textForCalculating.widthAnchor.constraint(equalTo: view.safeAreaLayoutGuide.widthAnchor, multiplier: 0.90),
+            textForCalculating.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -20),
+            textForCalculating.bottomAnchor.constraint(equalTo: stackViewFirstLine.topAnchor, constant: -20)
         ])
     }
+    
+    @objc func buttonActionDelete(_ sender: UIButton!) {
+        textForCalculating.text = "0"
+        firstNumber = 0
+        secondNumber = 0
+        currentOperation = .nothing
+    }
+    
+    @objc func buttonActionNumber(_ sender: UIButton!) {
+        let number = arrayOfNumbers.firstIndex(of: sender)!
+        
+        if textForCalculating.text == "0" {
+            textForCalculating.text = String(number)
+        } else {
+            textForCalculating.text! += String(number)
+        }
+        
+        if currentOperation == .nothing {
```

Comment:
Enum  лучше делать через switch а не if
```suggestion
        switch currentOperation  {
        case .nothing: code...
        default: break
        }
```